packages/react-components/src/components/DateInput.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { useRuleEngine, useFormIntl } from '@aemforms/af-react-renderer';
+import {State, FieldJson} from '@aemforms/af-core';
+import '../../dist/theme.css';
+
+const DateInput = (jsonfield: State<FieldJson>) => {
+    const [props, handlers] = useRuleEngine(jsonfield);
+    const { id, label, value, visible, errorMessage, valid, description, required, name } = props;
+    const i18n = useFormIntl();
+    const finalValue = value === undefined ? '' : value;
+    const validateState = valid === false ? 'invalid' : 'valid';
+    const error = validateState === 'invalid';
+    const formattedDescription = i18n.formatMessage({ id: `${description}` });
```

Comment:
This will not work. Have to test it?
You have to get the localization id from "props.properties['afs:translationIds']"

Reply:
Implemented!